Vizzuality/gfw
# Feature/public areas

## Overview

Allows users who are not logged in to view areas of interest from a share link.
- public share view for map area and dashboard
- area meta data hidden unless user logged in viewing own area
- change geodescriber to fetch based on language
- allows fetching and filtering of areas through selectors shared in areas provider
- remove subscribe modal
- fixes show, share and dashboard buttons in analysis

## Testing

- View the following area id: `598825ca6e68340010b582cd`, and you should see dashboards and map views without tags or meta data
- View you own areas as before
